<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fenMission</class>
 <widget class="QWidget" name="fenMission">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>268</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mission</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icones/notes.png</normaloff>:/icones/notes.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Navire:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QTextEdit" name="te_Operateurs">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>91</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QDateEdit" name="de_Fin"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Mission:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>51</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>1</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="le_Zone"/>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="le_Mission"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Zone:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="6">
    <widget class="QPushButton" name="btn_Annuler">
     <property name="text">
      <string>Annuler</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="btn_Valider">
     <property name="text">
      <string>Valider</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QDateEdit" name="de_Debut"/>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="le_ChefMission"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Chef de mission:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Opérateurs:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="le_Navire"/>
   </item>
   <item row="2" column="4">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Fin :</string>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QCheckBox" name="cb_Admin">
     <property name="text">
      <string>Mode Admin</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Début :</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
